src/actors/threaded_actor_injection.nit

```diff
@@ -0,0 +1,109 @@
+# This file is part of NIT ( http://www.nitlanguage.org ).
+#
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+# Injects model for the classes annotated with "is actor" so
+# that the later generated code can be properly used for compilation
+# This module uses the threaded actor model abstraction
+module threaded_actor_injection
```

Comment:
The diff between this module and `scheduled_actor_injection` looks like a `s/Actor/Proxy/`. You could probably use an abstraction here too...